File: WebApiModsen/frontend/src/app/events/event.model.ts
export interface EventModel {
    target: HTMLInputElement;
    id: number,
    title: string;
    description: string;
    dateOfEvent: string;
    location: string;
    categoryOfEvent: string;
    maximumOfMember: number;
    currentNumberOfMember: number;
    imageUrl: string;
  }

File: WebApiModsen/WebApiModsen.Core/Models/EventModel.cs
using WebApiModsen.WebApiModsen.Core.Enum;

namespace WebApiModsen.WebApiModsen.Core.Models
{
    public class EventModel
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public DateTime DateOfEvent { get; set; }
        public string Location { get; set; }
        public EventCategory CategoryOfEvent { get; set; }
        public int MaximumOfMember { get; set; }
        public int CurrentNumberOfMember { get; set; }
        public string ImageUrl { get; set; }
    }
}
